bin/metrics-es-cluster.rb

```diff
@@ -191,7 +191,7 @@ def run
       end
       acquire_cluster_metrics.each do |cluster_metric|
         cluster_metric[1].each do |k, v|
-          output(config[:scheme] + '.' + cluster_metric[0] + '.' + k, v)
+          output(config[:scheme] + '.' + cluster_metric[0] + '.' + k, v || 0)
```

Comment:
Just so I understand this graphite actually errors? If not I'd prefer this to be an opt in behavior via an option such as `--fill-missing-data`. Should there be an extra parenthesis like this: `output(config[:scheme] + '.' + cluster_metric[0] + '.' + k, (v || 0))`?